src/devices/src/virtio/block/device.rs

```diff
@@ -484,9 +489,69 @@ pub(crate) mod tests {
             &mut EventManager::new().unwrap(),
         );
         // Validate the queue operation finished successfully.
+        #[cfg(not(fuzzing))]
         assert_eq!(b.interrupt_evt.read().unwrap(), 1);
     }
 
+    #[cfg(fuzzing)]
+    pub fn fuzzing_func(data: &[u8]) {
```

Comment:
This function only ever consumes maximum (VQUEUES_NUM * 260) bytes from 'data: &[u8]'
is there a way to tell AFL to only provide slices of size [0, 260*VQUEUES_NUM]? so we don't
waste cycles for nothing?